tests/Operation/RenameCollectionFunctionalTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Driver\Exception\CommandException;
+use MongoDB\Operation\DropCollection;
+use MongoDB\Operation\FindOne;
+use MongoDB\Operation\InsertOne;
+use MongoDB\Operation\RenameCollection;
+use MongoDB\Tests\CommandObserver;
+
+use function version_compare;
+
+class RenameCollectionFunctionalTest extends FunctionalTestCase
```

Comment:
I think this file would do well to have a test for the `dropTarget` option. Perhaps two cases:

* Rename an existing collection when the target already exists and expect a server error of some kind.
* Repeat the first test but specify `dropTarget` as true and expect success.